src/sage/graphs/graph.py

```diff
@@ -4476,6 +4476,82 @@ def is_factor_critical(self, matching=None, algorithm='Edmonds', solver=None, ve
         # The graph is factor critical if all vertices are marked even
         return len(even) == self.order()
 
+    def is_valid_hom(G, H, hom):
+    r"""
+    Check if `hom` is a valid homomorphism from `G` to `H`.
+
+    INPUT:
+
+    - ``G`` -- a Sage graph
+
+    - ``H`` -- a Sage graph, as target of the homomorphism
+
+    - ``hom`` -- a graph homomorphism
+
+    OUTPUT:
```

Comment:
no need to explain that the output is a boolean. It's obvious from the name of the method.

Reply:
I try to keep `INPUT:` and `OUTPUT:` consistent for each function, even if it's evident from the function name.